Adding note regarding SARIF extension (#100299)
Azure/azure-content: articles/defender-for-cloud/azure-devops-extension.md

The pipeline will run for a few minutes and save the results.

> [!Note] Please ensure SARIF SAST Scans Tab extension is installed on the Azure DevOps organization to ensure the generated analysis results will be displayed automatically under the "Scans" tab.

Reviewer: ```suggestion
> [!Note] 
> Install the SARIF SAST Scans Tab extension on the Azure DevOps organization in order to ensure that the generated analysis results will be displayed automatically under the Scans tab.
```

Reviewer: @hkamel

Reviewer: @hkamel I need you to accept my revision so we can publish this PR
